tools/txg/TXG.hs

```diff
@@ -126,11 +126,12 @@ generateSimpleTransactions = do
             pure (a, b, operation)
           theCode = "(" ++ [op] ++ " " ++ show operandA ++ " " ++ show operandB ++ ")"
 
+      -- this contains the key of sender00
       kps <- testSomeKeyPairs
 
       let publicmeta = CM.PublicMeta
                        (CM.ChainId $ chainIdToText cid)
-                       ("sender" <> toText cid) 10 0.0001
+                       "sender00" 10 0.0001
```

Comment:
I just talked with Emmanuel about this in person.  He said he originally hard coded `sender00`.  So I think this is the right thing to do for now.  The only thing we would need to worry about is if `sender00` runs out of coins.

Also, `cid` is `ChainId`, so that appears to be just plain wrong.